cli/src/pcluster/cli/commands/dcv_connect.py

```diff
@@ -64,10 +64,11 @@ def _dcv_connect(args):
 
         try:
             url = _retry(_retrieve_dcv_session_url, func_args=[cmd, args.cluster_name, head_node_ip], attempts=4)
-            url_message = f"Please use the following one-time URL in your browser within 30 seconds:\n{url}"
+            url_message = f"Please use the following one-time URL in your browser within 30 seconds:"
```

Comment:
With this change we are removing an explicative message from stdout to make it easier to consume the url.
What if this command returns a JSON, such as:

```
{
    "message": "Please use the following one-time URL in your browser within 30 seconds",
    "url": "https://...."
}
```

This would be consistent with the other pcluster commands returning JSON and would make possible to consume the url easily with jq or, even better, with the --query option of the pcluster CLI.

What do you think?

Reply:
Sounds good to me. Please feel free to change it.